Backend/app/rider-platform/rider-app/Main/src/API.hs

```diff
@@ -49,7 +49,7 @@ type MainAPI =
     :<|> Beckn.API -- TODO :: Needs to be deprecated
     :<|> Beckn.APIV2
     :<|> MetroBeckn.API
-    :<|> ( Capture "merchantId" (ShortId DM.Merchant)
+    :<|> ( Capture "merchantOperatingCityId" (Id DMOC.MerchantOperatingCity)
```

Comment:
`merchant.shortId` is not the same as `merchantOperatingCity.id`, so it wouldn't backward compatible